uninstall/uninstall.sh

```diff
@@ -6,4 +6,196 @@ APP_PATH=`cd "$APP_PATH"; pwd`
 HOME_PATH=~
 INSTALL_PATH="${HOME_PATH}/.splashkit"
 
-rm -rf ${INSTALL_PATH}
+SKM_PATH=`cd "$APP_PATH/.."; pwd`
+
+HAS_PYTHON3=false
+
+source "${SKM_PATH}/tools/set_sk_env_vars.sh"
+
+echo "Attempting to uninstall SplashKit libraries"
+echo "This may require root access, please enter your password when prompted."
+echo
+
+if [ "$IS_WINDOWS" = true ]; then
+    PRIVILEGED=""
+else
+    PRIVILEGED="sudo"
+fi
+
+# Get directories for each OS
+echo "Detecting operating system..."
+if [ "$SK_OS" = "macos" ]; then
+    LIB_FILE="/usr/local/lib/libSplashKit.dylib"
+    LIB_DEST="/usr/local/lib"
+    INC_DEST="/usr/local/include"
+
+    # Check if python3 installed on macOS
+    if command -v python3 &> /dev/null && command -v brew &> /dev/null; then
```

Comment:
Again - this can move outside of the os detection so that finding out if you have python only occurs once.